src/themes/bankrate/theme.json

```diff
@@ -858,12 +858,7 @@
                 "mr": ["sm", "md"],
                 "verticalAlign": "middle"
               }
-            },
-            "border": "1px solid",
-            "borderColor": "dark-grey-03",
-            "borderRadius": "sm",
-            "py": ["sm", "md"],
-            "px": ["md", "lg"]
```

Comment:
Are we sure this code isn't being used somewhere else? on a content page for example. Might be worth searching in Contentful for this content model to see where it is used and make sure